src/structures/MessageEmbed.js

```diff
@@ -177,6 +168,15 @@ class MessageEmbed {
   }
 
   /**
+   * Convenience function for `<MessageEmbed>.addField('\u200B', '\u200B', inline)`.
+   * @param {boolean} [inline=false] Set the field to display inline
+   * @returns {MessageEmbed} This embed
+   */
+  addBlankField(inline = false) {
```

Comment:
~Why? Its such a useful small thing! Ain't nobody got time to get the unicode code for the 0-width space~